fe/fe-core/src/main/java/com/starrocks/external/hive/HiveColumnStats.java

```diff
@@ -161,6 +171,9 @@ public boolean init(String hiveType, ColumnStatisticsData statsData) {
                 LOG.warn("unexpected column type {}", hiveType);
                 break;
         }
+        if (isCompatible) {
```

Comment:
what's the meaning of `isCompatible` ?

Reply:
@gengjun-git

Reply:
isCompatible means the stats data is valid

Reply:
So why use `Compatible` to mean `valid`?  I think which is incomprehensible.